dissect/target/plugins/os/unix/services.py

```diff
@@ -106,43 +107,45 @@ def should_ignore_file(needle: str, haystack: list) -> bool:
     return False
 
 
-def parse_systemd_config(fh: BinaryIO) -> str:
+def parse_systemd_config(fh: TextIO) -> str:
     """Returns a string of key/value pairs from a toml/ini-like string.
 
     This should probably be rewritten to return a proper dict as in
     its current form this is only useful when used in Splunk.
     """
-    variables = ""
+    parser = ConfigParser(strict=False, delimiters=("=",), allow_no_value=True)
```

Comment:
```suggestion
    parser = ConfigParser(strict=False, delimiters=("=",), allow_no_value=True)
```

`allow_no_value=True` is not realy what I meant, as that would allow

```
[Unit]
test
```
which I don't think is allowed in systemd configs (never tested it so I'm not sure). I was referring to the case of:

```
[Unit]
test=
```
Maybe this is already handled by the default settings.